image_client/client_tools.py

```diff
@@ -43,15 +47,15 @@ def parse_command_line():
     return parser.parse_args()
 
 
-
 def main(args):
 
     if args.subcommand == 'search':
         image_client = ImageClient()
     elif args.subcommand == 'import':
-
         if args.collection == "Botany":
-             BotanyImporter()
+            BotanyImporter(paths=botany_importer_config)
+        elif args.collection == 'PIC':
```

Comment:
let's change this to Botany_picturae, as we might have other collections use picturae eventually.